### DISCUSSION TITLE
2021.06.11

### DISCUSSION THREAD

**Discussion by Settpark** [malaheaven/issue-tracker#37] [General]
iOS: OAuth 구현했습니다. Issue작성 UI 만들었습니다. iOS는 오늘 행복합니다.

**Comment by 17-sss** [comment]
**FE**

오늘도 열시미 검색을 하다가 방법을 찾아내어 적용했습니다!
빨리 구현하려다보니까 리팩이 필요한 코드가 나와버렸네요...ㅎ
**_행복합니다 오늘도_**
~~살려주세요.. 놋북에서 파란 화면을 두번이나봤어요~~

**Comment by malaheaven** [comment]
## BE
금일은 jwt를 적용했습니다
